src/tests/frontend/index.html

```diff
@@ -8,6 +8,7 @@
   </head>
   <body>
     <div id="console"></div>
+    <div id="debuglogs"></div>
```

Comment:
Ha, good question. Actually my wishful thinking would be to link browser console output, test result and server console output together. At the moment it is rather hard to interpret the outputs, because multiple tests run concurrently and there is a 5 second delay between server logs and console dumps.

I assume there was no reason except I didn't want the test results (PASS/FAIL and trace) to be mixed with the debug logs. For those runners, that have a console.json output, we can just call `console.log(...)`. Now that I write this I think I could have used manual saucelabs runner, but I usually like the work flow of pushing to github and crawling the logs more then manually run sauce connect and connecting manually.

Maybe entirely revert this additional debug log functionality?

Reply:
Yeah, I say revert it and reuse the existing logging.